source/elements/oneDNN/source/introduction.rst

```diff
@@ -137,4 +137,15 @@ a boolean ``allow_empty`` parameter that defaults to ``false`` and that makes
 the library to return an empty object (a zero object in case of memory
 descriptors) when an object cannot be constructed instead of throwing an error.
 
+**********
+Namespaces
+**********
+
+All oneDNN functions and classes reside in ``::dnnl`` namespace. The functions
+that accept or return DPC++ objects such as command queues or buffers reside in
+``::dnnl::sycl_interop`` namespace.
+
+Furthermore, oneDNN defines ``::oneapi::dnnl`` namespace, that is an alias for
+the ``::dnnl`` namespace.
```

Comment:
Do we have to have `::oneapi::dnnl::sycl_interop` too? If so, we may want to mention it explicitly.